.github/ISSUE_TEMPLATE.md

```diff
@@ -1,24 +1,28 @@
-Thanks for submitting an issue! 
+Thanks for submitting an issue! Please fill in as much of the template below as
+you can.
 
--------------BUG REPORT --------------------
-1. Fill in as much of the template below as you can. 
+------------- BUG REPORT TEMPLATE --------------------
 
-2. What kops version are you running? use `kops version`
+1. What kops version are you running? Use `kops version`?
 
-3. What Kubernetes version are you running? use `kubectl version`
+1. What Kubernetes version are you running? Use `kubectl version`?
 
-4. What cloud provider are you using?
+1. What cloud provider are you using?
 
-5. What commands did you execute (Please provide cluster manifest `kops get --name my.example.com`, if available) and what happened after commands executed?
+1. What commands did you execute, and what happened after the commands executed?
 
-6. What you expected to happen:
+1. Can you provide cluster manifest `kops get --name my.example.com -oyaml`?
 
-7. How can we to reproduce it (as minimally and precisely as possible):
+1. What you expected to happen?
```

Comment:
Your numbers got reset here. Silly markdown!

What did you expect to happen when you ran this? What would you like this command to do?

Reply:
If you just do 1. 1. 1. you do not have to worry about changing them ever.  The render with the correct numbers ;)